docs/docs/dev_docs/contracts/portals/main.md

```diff
@@ -47,6 +47,19 @@ Computing the `content` must be done manually in its current form, as we are sti
 
 #include_code claim_public /yarn-project/noir-contracts/src/contracts/token_bridge_contract/src/main.nr rust
 
+:::info
+The `content_hash` is a sha256 truncated to a field element (~ 254 bits). Same is true for the secretHash that gets passed on L1
```

Comment:
When you are saying same, I'm understanding it as the sha256 and then truncated, but that is not the case, the secretHash is a different hash function.